### DISCUSSION TITLE
Searchbar in header Animation

### DISCUSSION THREAD

**Discussion by dhanushm360** [software-mansion/react-native-screens#901] [Q&A] [answered]
Is there a way where you can implement this ios search bar animation with react-native screens? When the searchbar is focused on, it becomes the header. 

https://user-images.githubusercontent.com/49541102/116464950-32db5c00-a832-11eb-885c-ae3a5dd7870b.mov

This is what I have right now:

https://user-images.githubusercontent.com/49541102/116465323-b432ee80-a832-11eb-9fd9-2e888cfc4aab.mov

**Comment by pouyarezvani** [comment]
@dhanushm360 Are you currently using Expo for this project? 
Or are you using the bare react native app?

**Reply by dhanushm360** [reply]
@pouyarezvani I am using Expo for this project.

**Comment by kacperkapusciak** [comment, marked as answer]
Hi @dhanushm360 
we've recently added search bar implementation in `react-native-screens` in v3.2.0 that does exactly that:

https://user-images.githubusercontent.com/39658211/117790966-24d20600-b24a-11eb-9ce9-e22d112ee351.mov

It available out-of-the-box in the bare workflow.

In an Expo managed workflow it's a bit trickier but I think it's doable with EAS Build. Check out [this article](https://blog.expo.io/expo-managed-workflow-in-2021-d1c9b68aa10) for more information.

The newest version of `screens` (hopefully) will be included in Expo SDK 42.
